Client/Frontend/AuthenticationManager/PasscodeViews.swift

```diff
@@ -162,6 +163,9 @@ class PasscodePane: UIView {
             make.bottom.equalTo(centerContainer)
             make.size.equalTo(PasscodeUX.PasscodeFieldSize)
         }
+        
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(PasscodePane.keyboardWillHide(_:)), name: UIKeyboardWillHideNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(PasscodePane.keyboardWillShow(_:)), name: UIKeyboardWillShowNotification, object: nil)
```

Comment:
Whenever registering observers in `init`, you'll want to pair it up with a `removeObserver` call in the `deinit` method so it doesn't linger after the VC is gone.